<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reprodutor</class>
 <widget class="QWidget" name="Reprodutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>480</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: none;
background: none;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="12" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGraphicsView" name="graficosVideo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>350</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>59</green>
        <blue>59</blue>
       </color>
      </brush>
     </property>
     <property name="foregroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>59</red>
        <green>59</green>
        <blue>59</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_voltarVideo">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/anterior/anterior.png&quot;);
	background-color: none;
	border-radius: 2px;
	border: none;
	min-width: 60px;
	min-height: 50px;
	font-weight: bold;
}

QPushButton:hover {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/anterior/anterior-hover.png&quot;);
	background-color: none;
}

QPushButton:pressed {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/anterior/anterior-hover.png&quot;);
	background-color: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_video_tocar">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/pause/pause.png&quot;);
	background-color: none;
	border-radius: 2px;
	border: none;
	min-width: 60px;
	min-height: 50px;
	font-weight: bold;
}

QPushButton:hover {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/pause/pause-hover.png&quot;);
	background-color: none;
}

QPushButton:pressed {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/pause/pause-hover.png&quot;);
	background-color: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="pushButton_passarVideo">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/proximo/proximo.png&quot;);
	background-color: none;
	border-radius: 2px;
	border: none;
	min-width: 60px;
	min-height: 50px;
	font-weight: bold;
}

QPushButton:hover {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/proximo/proximo-hover.png&quot;);
	background-color: none;
}

QPushButton:pressed {
	border-image: url(&quot;:/icones/assets/icones/reprodutor/proximo/proximo-hover.png&quot;);
	background-color: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_volume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-image: url(&quot;:/icones/assets/icones/som/com-som.png&quot;);
	background-color: none;
	border-radius: 2px;
	border: none;
	min-width: 40px;
	min-height: 40px;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_volume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider{
border-image: none;
background: none;
}

QSlider::handle:horizontal {
 	background: #4466c3;
	border: 1px solid #4466c3;
	border-radius: 7px;
	width: 15px;
	height: 15px;
	margin: -10px 0;
}

QSlider::sub-page:horizontal {
 	background: #ffb32d;
 }</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_tempo_atual">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-weight: bold;
border-image: none;
background: none;</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_tempo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QSlider{
	border-image: none;
background: none;
}

QSlider::handle:horizontal {
 	background: #4466c3;
	border: 1px solid #4466c3;
	border-radius: 7px;
	width: 15px;
	height: 15px;
	margin: -10px 0;
}

QSlider::sub-page:horizontal {
 	background: #ffb32d;
 }</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_tempo_final">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: black;
font-weight: bold;
border-image: none;
background: none;</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
